index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+       <meta charset="UTF-8" />
+       <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+       <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+       <link rel="stylesheet" href="style.css" />
+       <title>My Mobile Portfolio</title>
+    </head>
+    <body>
+        <header>
+                <div class=""></div>
+                <div class=""></div>
+                
+                <nav class="content">
+                    <a href="#" class="nav-branding">Bantealem</a>
+                
+                    <ul class="nav-menu">
+                        <li class="nav-item">
+                            <a href="#" class="nav-link">Mobile Portfolio</a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="#" class="nav-link">About</a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="#" class="nav-link">Contacts</a>
+                        </li>
+                    </ul>
+                    <div class="content2">
+                        <span class="bar"></span>
+                        <span class="bar"></span>
+                        <span class="bar"></span>
+                    </div>
```

Comment:
- [ ] [OPTIONAL] You can export the whole hamburger menu icon from Figma. So, please export the whole icon and use it instead of building the icon in CSS.